--- coregrind/m_debuginfo/d3basics.c	2018-07-25 16:12:00.311313468 +0800
+++ coregrind/m_debuginfo/d3basics.c.new	2018-07-25 15:50:40.299334557 +0800
@@ -920,10 +920,10 @@
                     "does not terminate expression");
             break;
          default:
-            if (!VG_(clo_xml))
-               VG_(message)(Vg_DebugMsg, 
-                            "warning: evaluate_Dwarf3_Expr: unhandled "
-                            "DW_OP_ 0x%x\n", (Int)opcode); 
+            //if (!VG_(clo_xml))
+            //   VG_(message)(Vg_DebugMsg, 
+            //                "warning: evaluate_Dwarf3_Expr: unhandled "
+            //                "DW_OP_ 0x%x\n", (Int)opcode); 
             FAIL("evaluate_Dwarf3_Expr: unhandled DW_OP_");
             /*NOTREACHED*/
       }
